README.rst

```diff
@@ -75,18 +75,20 @@ To use Octopus after installation
 3. Use octopus (it should be in the ``$PATH``). You can check the octopus version using ``octopus version``.
 
 
-Compile Octopus on Debian Linux (without spack)
-===============================================
+Octopus in Docker container
+===========================
 
-|debian-octopusstable| Debian Bullseye (11), Last octopus release (11.4)
+See https://github.com/fangohr/octopus-in-docker .
 
-|debian-octopusdevelop| Debian Bullseye (11), Octopus develop branch
 
+Compile Octopus on Debian Linux (without spack)
+===============================================
 
-Octopus in Docker container
-===========================
+See for example the steps in
+`this Dockerfile <https://github.com/fangohr/octopus-in-docker/blob/main/Dockerfile>`__
```

Comment:
I feel the sections `Octopus in Docker container` and `Compile Octopus on Debian Linux (without spack)` are similar, instead of two headings, should we maybe combine them?

Reply:
If one is looking for how to compile Octopus on Linux, then the one heading is the one you search. If you look for Octopus in Docker, you are looking for the other one. You can see both in the table of contents. Let's leave it for now - but look over it again critically later; perhaps we also get user feedback.